.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Close stale PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  contents: write
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-pr-message: 'Hi there! This PR has not been updated for 60 days, and will be closed in 14 days. To keep it open, please update the PR.'
+          days-before-stale: 60
+          days-before-close: 14
+          days-before-issue-stale: -1
+          exempt-pr-labels: keep-open
+          start-date: '2023-12-05T00:00:00Z'
+          delete-branch: true
```

Comment:
I guess this just doesn't happen if it's a PR from a fork.

Reply:
indeed. [the fix](https://github.com/actions/stale/commit/c4a13d8dca059f5b6e32f5cb631f621870106178) was part of v8, so we should have it included